client/ayon_maya/plugins/publish/validate_rendersettings.py

```diff
@@ -237,7 +237,10 @@ def get_invalid(cls, instance):
                     dir_prefix))
 
         if renderer == "arnold":
-            multipart = cmds.getAttr("defaultArnoldDriver.mergeAOVs")
+            multipart = (
+                cmds.getAttr("defaultArnoldDriver.mergeAOVs") or
+                cmds.getAttr("defaultArnoldDriver.multipart")
```

Comment:
I vaguely recall enabling multipart didn't actually _enforce_ AOVs to be merged - it would just be 'irrelevant' if enabled but not merging parts? Or? Hence, multipart being enabled doesn't actually enforce it results in a single file which as far as I know is what we use the `multipart` variable for? Hence, the question - is this change actually correct?

Reply:
afaik yes,  Multipart  is not tied to be used only with multilayered EXR and been completely separate format (neither supported in single layered EXR nor multilayered one) so probably not needed...